eng/common/scripts/stress-testing/deploy-stress-tests.ps1

```diff
@@ -131,13 +136,13 @@ function DeployStressPackage(
             $imageTag = "${registry}.azurecr.io/$($repository.ToLower())/$($imageName):$deployId"
             Write-Host "Building and pushing stress test docker image '$imageTag'"
             Run docker build -t $imageTag -f $dockerFile.FullName $dockerFile.DirectoryName
-            if ($LASTEXITCODE) { return $LASTEXITCODE }
+            if ($LASTEXITCODE) { return }
```

Comment:
Do you want to make errors a no-op now?

Reply:
I realized this returned exit code isn't used anywhere. The `Run` function will handle adding the failures to a failures array which is then checked and reported after all stress tests have been deployed. I still want the `return` statements for control flow to stop execution on a per-test basis.